### DISCUSSION TITLE
Ability to change tag case to remove duplicates

### DISCUSSION THREAD

**Discussion by ilc168** [windingwind/zotero-actions-tags#157] [Ideas]
I have many duplicate tags in my zotero library that are based on capitalisation. e.g. Water Supply, water supply, and Water supply; would appear 3 times. I'd love a feature that could automate changing all tag cases to lower case. 

While, I'm here, a feature that compared  similar tags and gave options to merge them would be handy too. e.g. "water supply" and "fresh water supply"
Thanks for your work on this.

**Comment by windingwind** [comment]
This is what you need: https://github.com/windingwind/zotero-actions-tags/discussions/113

**Comment by ilc168** [comment]
I'm not sure how I would modify this script do a bulk capitalisation change? It looks like the script changes one tag for another (noting I'm not savvy with scripts).

**Comment by windingwind** [comment]
#203
